string station
int32 objects
---
int32 objOnTable
